Marlin/src/lcd/extui/lib/mks_ui/tft_Language_en.h

```diff
@@ -275,6 +276,7 @@
 #define PRINT_TEXT_EN           "Printing"
 #define EXTRUDE_TEXT_EN         "Extrusion"
 #define LEVELING_TEXT_EN        "Leveling"
+#define MLEVELING_TEXT_EN       "Leveling"
```

Comment:
Is there a reason for these to be unique constants, even though they have identical contents?

The same question applies to `TOOL_MLEVELING_EN`.

Reply:
Maybe all four of these should be one constant?

Reply:
M seems to stand for `manual`. They are the same for English, but could not be the same for other lang. Anyway, I think gcc will take care of this for us.

Reply:
#define MLEVELING_TEXT_EN       "Leveling" 
It just needs to be removed. This is the tail from my merge request on the MKS branch. In this code it is not used and will not be used.

Reply:
![11](https://user-images.githubusercontent.com/52313714/101644587-6dd92080-3a46-11eb-8138-6ab73e7ca4e5.jpg)
![22](https://user-images.githubusercontent.com/52313714/101644590-6f0a4d80-3a46-11eb-8c40-d3aa182f03fa.jpg)
all this is not used and should be removed